File: src/HostelSystem.Domain/Gosc.cs
using System;

namespace HostelSystem.Domain
{
    public class Gosc : CommonEntity
    {
        public string Imie { get; set; }
        public string Nazwisko { get; set; }
        public string Email { get; set; }
        public Guid? RezerwacjaId { get; set; }
        public string Telefon { get; set; }
        public string Adres { get; set; }

        public Rezerwacja Rezerwacja { get; set; }
    }
}

File: src/HostelSystem.Web/Sencha/app/model/Gosc.js
Ext.define('HostelSystem.Web.model.Gosc', {
    extend: 'Ext.data.Model',

    fields: [
        { name: 'imie', type: 'string' },
        { name: 'nazwisko', type: 'string' },
        { name: 'email', type: 'string' },
        { name: 'telefon', type: 'string' },
        { name: 'adres', type: 'string' }
    ],

    belongsTo: 'HostelSystem.Web.model.Rezerwacja'
});